examples/2-query/src/GetAll.re

```diff
@@ -0,0 +1,89 @@
+/* Opens our styles module */
+open GetAllStyles;
+
+type state = {
+  listOfPokemons: Js.Array.t(string),
```

Comment:
Can you talk about your decision to use `Js.Array.t`? I'm not super familiar with it, and IIRC Reason's native `array` will compile to a JS array directly, so you're safe binding to it in that way. A positive I can think of is that you wouldn't need to convert the array to a list to do the filtering below (since the native `Array` module doesn't have a `filter` operation). Happy to keep this as is, mostly just curious about your thinking!

Reply:
No decision behind this, I was just scrambling at stuff 🤷‍♀️ I'd rather rely on as much 'native' reason bindings as possible so have gone back to Array. Sucks having to convert to List but eh.